x-pack/test_serverless/api_integration/test_suites/common/reporting/generate_csv_discover.ts

```diff
@@ -11,7 +11,7 @@ import type { JobParamsCSV } from '@kbn/reporting-export-types-csv-common';
 import type { Filter } from '@kbn/es-query';
 import { FtrProviderContext } from '../../../ftr_provider_context';
 
-export default ({ getService }: FtrProviderContext) => {
+export default function ({ getService }: FtrProviderContext) {
```

Comment:
Had to do this to address the following CI [failure](https://buildkite.com/elastic/kibana-pull-request/builds/199602#018e7da3-160a-4ca7-a662-357656df9150):
<img width="1108" alt="Screenshot 2024-03-27 at 13 04 23" src="https://github.com/elastic/kibana/assets/2748376/52bccd41-e4e7-43cb-a0f3-061742b400ab">

Also [this](https://buildkite.com/elastic/kibana-pull-request/builds/199602#018e7dbf-a6eb-4386-8925-51b79e136538):
<img width="1057" alt="image" src="https://github.com/elastic/kibana/assets/2748376/88bcb573-1600-4c66-b704-8cd68c168c76">

cc: @wayneseymour